index.py

```diff
@@ -55,18 +48,20 @@ def POST(self, crap):
         return handler()
 
 
-def get_available_layers((lon, lat)):
+def get_available_layers(lon_lat):
+    lon, lat = lon_lat
     layers = ["osm"]
     return layers
 
 
-def geocoder_describe((lon, lat), zoom, locale):
+def geocoder_describe(lon_lat, zoom, locale):
+    lon, lat = lon_lat
     descr = ()
     # try:
     if locale == "en":
-        namestring = """COALESCE("name:en","int_name", replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(translate("name",'абвгдезиклмнопрстуфьАБВГДЕЗИКЛМНОПРСТУФЬ','abvgdeziklmnoprstuf’ABVGDEZIKLMNOPRSTUF’'),'х','kh'),'Х','Kh'),'ц','ts'),'Ц','Ts'),'ч','ch'),'Ч','Ch'),'ш','sh'),'Ш','Sh'),'щ','shch'),'Щ','Shch'),'ъ','”'),'Ъ','”'),'ё','yo'),'Ё','Yo'),'ы','y'),'Ы','Y'),'э','·e'),'Э','E'),'ю','yu'),'Ю','Yu'),'й','y'),'Й','Y'),'я','ya'),'Я','Ya'),'ж','zh'),'Ж','Zh')) AS name"""
+        namestring = """COALESCE(tags->'name:en',tags->'int_name', replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(replace(translate("name",'абвгдезиклмнопрстуфьАБВГДЕЗИКЛМНОПРСТУФЬ','abvgdeziklmnoprstuf’ABVGDEZIKLMNOPRSTUF’'),'х','kh'),'Х','Kh'),'ц','ts'),'Ц','Ts'),'ч','ch'),'Ч','Ch'),'ш','sh'),'Ш','Sh'),'щ','shch'),'Щ','Shch'),'ъ','”'),'Ъ','”'),'ё','yo'),'Ё','Yo'),'ы','y'),'Ы','Y'),'э','·e'),'Э','E'),'ю','yu'),'Ю','Yu'),'й','y'),'Й','Y'),'я','ya'),'Я','Ya'),'ж','zh'),'Ж','Zh')) AS name"""
     else:
-        namestring = 'COALESCE("name:' + locale + '", "name") AS name'
+        namestring = 'COALESCE(tags->\'name:' + locale + '\', "name") AS name'
```

Comment:
really, there should be a custom osm2pgsql style lying around.

Reply:
reverted with adding https://github.com/Komzpa/furry-sansa/blob/master/assets/osm2pgsql.style config